File: 프로그래머스/0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.java
import java.util.Arrays;
class Solution {
    public int[] solution(String my_string) {
        my_string = my_string.replaceAll("[a-zA-Z]", "");
        int[] result = new int[my_string.length()];

        for(int i = 0; i <= my_string.length() - 1; i++){
            result[i] = Character.valueOf(my_string.charAt(i))-48;
        }

        Arrays.sort(result);

        return result;
    }
}

File: 프로그래머스/0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.js
function solution(my_string) {
    // 알파벳을 제외한 모든 문자 필터링
    let filteredString = my_string.replace(/[a-zA-Z]/g, '');

    // 남은 문자들을 숫자로 변환
    let result = Array.from(filteredString).map(char => parseInt(char));

    // 결과를 정렬
    result.sort((a, b) => a - b);

    return result;
}
